src/test/java/ladder/domain/LadderTest.java

```diff
@@ -10,7 +10,7 @@
 
 public class LadderTest {
 
-    @DisplayName("Ladder는 height과 width를 가진다.")
+    @DisplayName("height과 width로 Ladder 생성")
```

Comment:
성공 테스트일테니 
```suggestion
@DisplayName("유효한 height과 width로 Ladder 생성할 경우 정상적으로 Ladder 인스턴스가 생성된다.")
```
이렇게 작성하면 좀 더 가독성이 올라가겠죠